sklearn/datasets/_covtype.py

```diff
@@ -80,7 +100,13 @@ def fetch_covtype(*, data_home=None, download_if_missing=True,
         If True, returns ``(data.data, data.target)`` instead of a Bunch
         object.
 
-        .. versionadded:: 0.20
+    as_frame : bool, default=False
```

Comment:
This is the part that's missing in the other PR.

Reply:
Got it, thanks